client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -178,7 +178,8 @@ const defineMonacoThemes = (
     inherit: true,
     // TODO: Use actual color from style-guide
     colors: {
-      'editor.background': options.usesMultifileEditor ? '#eee' : '#fff'
+      'editor.background': options.usesMultifileEditor ? '#eee' : '#fff',
+      'editor.lineHighlightBorder': '#002ead'
```

Comment:
```suggestion
      'editor.lineHighlightBorder': '#cee8fc'
```

Reply:
Also let's set the border color for the vs-dark-custom theme to :`#0e4470`

Other than that this is LGMT